task-list/app/models/person.rb

```diff
@@ -0,0 +1,8 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def uncompleted
```

Comment:
Since this method returns the count, it might make sense to change the name of the method to something like `uncompleted_count` since it returns the length rather than the tasks themselves